content/docs/projects/trust.md

```diff
@@ -48,14 +48,27 @@ spec:
       -----END CERTIFICATE-----
   target:
     # Data synced to the ConfigMap `my-org.com` at the key `root-certs.pem` in
-    # every namespace.
+    # every namespace that has the label "linkerd.io/inject=enabled".
     configMap:
       key: "root-certs.pem"
+    namespaceSelector:
+      matchLabels:
+        linkerd.io/inject: "enabled"
 ```
 
 Bundle currently supports the source types `configMap`, `secret` and `inLine`,
 and target type `configMap`.
 
+#### Namespace Selector
+
+The target `namespaceSelector` can be used for scoping which Namespaces targets
+are synced to, supporting the field `matchLabels`. Please see
+[here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
+for more information and how label selectors are configured.
+
+If `namespaceSelector` is empty, a bundle target will be synced to all
+Namespaces.
```

Comment:
suggestion (non-blocking): a rewording which maybe reads a little smoother and voids a link called "here"; what do you think?

```suggestion
The `namespaceSelector` field in a target reduces the set of Kubernetes namespaces to which that target is synced.

Label selectors are detailed in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors).

If `namespaceSelector` is empty, a bundle target will be synced to all namespaces.
```